File: BackEnd/src/main/java/org/example/dto/MecanicoDTO.java
package org.example.dto;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MecanicoDTO {
    private Long idMecanico;
    private String nombre;
    private String email;
    private Integer cantidadVehiculos;
}

File: Android/app/src/main/java/com/example/autolog/core/data/remote/dto/mecanico/MecanicoDTO.kt
package com.example.autolog.core.data.remote.dto.mecanico

import com.google.gson.annotations.SerializedName

data class MecanicoDTO(
    @SerializedName("idMecanico")
    val idMecanico: Long,
    
    @SerializedName("nombre")
    val nombre: String,
    
    @SerializedName("email")
    val email: String,
    
    @SerializedName("cantidadVehiculos")
    val cantidadVehiculos: Int
)
